x-pack/plugins/secops/public/components/timeline/helpers.tsx

```diff
@@ -78,21 +75,6 @@ export const combineQueries = (
   };
 };
 
-export const getEventNotes = ({
-  eventIdToNoteIds,
-  notesById,
-}: {
-  eventIdToNoteIds: { [eventId: string]: string[] };
-  notesById: NotesById;
-}): { [eventId: string]: Note[] } =>
-  Object.keys(eventIdToNoteIds).reduce(
-    (acc, eventId) => ({
-      ...acc,
-      [eventId]: getApplicableNotes({ noteIds: eventIdToNoteIds[eventId], notesById }),
-    }),
-    {}
```

Comment:
Will keep a lookout in future PR's for unintentional reference updates like this being used in `render()` 👍